docs/Guides.md

```diff
@@ -0,0 +1,7 @@
+# OpenVINO&trade; Model Server Guides
+
+## Introduction
+This document gives information about various guides used in OpenVINO&trade; Model Server
+
+- [Model Server Developer Guide](./DeveloperGuide.md)
+- [Installation Guide](./InstallationGuides.md)
```

Comment:
Installation guides includes just 3 links. I wonder if it could be combined. I would expected there will be far less developers and contributors from users so maybe such guide doesn't need to be on the top of the list.

Reply:
Done.